README.md

```diff
@@ -118,6 +118,7 @@ Quick links:
 
 *   [Meeting notes](https://docs.google.com/document/d/1CWUAkBrcm9KPclAu8fWHZzByZ0yhsQdRggnEdqtRMQ8/edit)
 *   [Zoom meeting link](https://zoom.us/my/cncftikvproject)
+*   [Add to my calendar](https://calendar.google.com/calendar/r/month/2020/4/23?eid=MXZiaDg4YmEwYW1rbDlwajRtdjIyb2w3ZTlfMjAyMDA0MjNUMDEwMDAwWiBsaW51eGZvdW5kYXRpb24ub3JnX281YXZqbHZ0MmNhZTlicTdhOTVlbWM0NzQwQGc&ctz=America/New_York&sf=true)
```

Comment:
Can the link be a regular one instead of editing from time to time?

Reply:
This is generated from the CNCF. And also it makes more sense to add to calendar only for the current week because we can't make the decision on behalf of other people.

Reply:
Then I don't think it's necessary to put the scheduling information in README, maybe we need to put it on a web page.